### PR TITLE
Add configuration for ready-to-go downloaded sample - Android

### PR DESCRIPTION
**Pull request by archieag**
#sign-off 
Please merge

## Purpose

* ...
Adding configuration for this quickstart code sample to be ready to run with app configuration in the portal.

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Comment by archieag** [comment]
@rpdome Please review and merge

File: AppCreationScripts/apps.json [hunk 1]
```diff
@@ -27,7 +27,34 @@
...
           ]
         }
-      ]
-    }
+      ],
+      "codeConfigurations": [	
+          {
+            "settingFile": "/app/src/main/res/raw/auth_config_multiple_account.json",
+            "replaceTokens": {
+                "appId": "0984a7b6-bc13-4141-8b0d-8f767e136bb7",
```
Comment (hamiltonha) [inline AppCreationScripts/apps.json]: Should this be replaced with the users information? "Enter_the_application_Id_here"? Ditto for redirect uri, package name, and key hash

Comment (rpdome) [reply AppCreationScripts/apps.json]: I'm guessing this might means "replace these key pair values", but I'll let @archieag to confirm.

Comment (archieag) [reply AppCreationScripts/apps.json]: This config informs portal what values to replace. For ex, it'll know that appId needs to be updated in this settings file. And, then, it'll look the value of this appId and replace with one in the portal.
So, it needs to match what's in the settings file.

**Pull request merged**